source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -371,6 +373,158 @@ namespace
         return prim;
     }
 
+    PvtPrim* readCollection(const CollectionPtr& src, PvtPrim* parent, PvtStage* stage)
+    {
+        const RtToken name(src->getName());
+
+        PvtPrim* prim = stage->createPrim(parent->getPath(), name, RtCollection::typeName());
+        RtCollection collection(prim->hnd());
+        collection.getIncludeGeom().setValueString(src->getIncludeGeom());
+        collection.getExcludeGeom().setValueString(src->getExcludeGeom());
+
+        // createCollectionConnections() handles creating collection inclusion
```

Comment:
Is this comment is misplaced?

Reply:
It's supposed to indicate that this functionality is not done here but in the order function.

Reply:
I see